spec/index.spec.ts

```diff
@@ -86,14 +86,17 @@ describe('axiosRetry(axios, { retries, retryCondition })', () => {
 
         client.get('http://example.com/test').then((result) => {
           expect(result.status).toBe(200);
+          // @ts-ignore
```

Comment:
IMO it would be better to use non-null assertion operator (`!`) instead of adding `ts-ignore` as that hides all issues on the line.

Can just do:

```ts
expect(result.config[namespace]!.retries).toBe(1);
```